kolibri/utils/file_storage.py

```diff
@@ -0,0 +1,9 @@
+from django.conf import settings
+from storages.backends.gcloud import GoogleCloudStorage  # noqa
+
+# https://django-storages.readthedocs.io/en/latest/backends/gcloud.html#google-cloud-storage
+
+
+class KolibriFileStorage(GoogleCloudStorage):
```

Comment:
I think here we can override the [default values ](https://docs.djangoproject.com/en/5.1/ref/files/storage/#django.core.files.storage.FileSystemStorage.location) for the location if needed

Reply:
Not clear to me why this file is necessary - we can just directly use the GoogleCloudStorage class and use settings (with their own options in options.py to allow them to be defined).

https://django-storages.readthedocs.io/en/latest/backends/gcloud.html#settings